Removed unsupported versions from database products (#5764)
EnterpriseDB/docs: product_docs/docs/epas/14/installing/linux_install_details/managing_an_advanced_server_installation.mdx

> For more information about using the `pg_ctl` utility, or the command line options available, see the official PostgreSQL Core Documentation available at:

Reviewer: ```suggestion
For more information about using the `pg_ctl` utility or the command-line options available, see the official [PostgreSQL Core Documentation](https://www.postgresql.org/docs/current/static/app-pg-ctl.html). 
```

> [https://www.postgresql.org/docs/current/static/app-pg-ctl.html]<https://www.postgresql.org/docs/current/static/app-pg-ctl.html>

Reviewer: ```suggestion
```

> `edb-psql` is a symbolic link to a binary called `psql`, a modified version of the PostgreSQL community `psql`, with added support for Advanced Server features. For more information about using the command line client, see the PostgreSQL Core Documentation at:

Reviewer: ```suggestion
`edb-psql` is a symbolic link to a binary called `psql`, a modified version of the PostgreSQL community `psql`, with added support for Advanced Server features. For more information about using the command line client, see the [PostgreSQL Core Documentation](https://www.postgresql.org/docs/current/static/app-psql.html).
```

> [https://www.postgresql.org/docs/current/static/app-psql.html]<https://www.postgresql.org/docs/current/static/app-psql.html>

Reviewer: ```suggestion
```

### Managing authentication on a Debian or Ubuntu host

By default, the server is running with the peer or md5 permission on a Debian or Ubuntu host. You can change the authentication method by modifying the `pg_hba.conf` file, located under `/etc/edb-as/14/main/`.

> For more information about modifying the `pg_hba.conf` file, see the PostgreSQL core documentation available at:

Reviewer: ```suggestion
For more information about modifying the `pg_hba.conf` file, see the [PostgreSQL core documentation](https://www.postgresql.org/docs/current/auth-pg-hba-conf.html). 
```

> [https://www.postgresql.org/docs/current/auth-pg-hba-conf.html]<https://www.postgresql.org/docs/current/auth-pg-hba-conf.html>

Reviewer: ```suggestion
```

The PostgreSQL `initdb` command creates a database cluster; when installing EDB Postgres Advanced Server with an RPM package, the `initdb` executable is in `/usr/edb/asx.x/bin`. After installing EDB Postgres Advanced Server, you must manually configure the service and invoke `initdb` to create your cluster. When invoking `initdb`, you can:

-   Specify environment options on the command line.
-   Include the `systemd` service manager on RHEL/Rocky Linux/AlmaLinux 8.x and use a service configuration file to configure the environment.

> To review the `initdb` documentation, visit:

Reviewer: ```suggestion
For more information, see the [initdb documentation](https://www.postgresql.org/docs/current/static/app-initdb.html).
```

> [https://www.postgresql.org/docs/current/static/app-initdb.html]<https://www.postgresql.org/docs/current/static/app-initdb.html>

Reviewer: ```suggestion
```

> For more information about using `initdb`, and the available cluster configuration options, see the PostgreSQL Core Documentation available at:

Reviewer: ```suggestion
For more information about using `initdb`, and the available cluster configuration options, see the [PostgreSQL Core Documentation](https://www.postgresql.org/docs/current/static/app-initdb.html).
```

> [https://www.postgresql.org/docs/current/static/app-initdb.html]<https://www.postgresql.org/docs/current/static/app-initdb.html>

Reviewer: ```suggestion
```
